Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -151,6 +151,7 @@ public void Initialize(TelemetryConfiguration configuration)
                             if (this.IncludeDiagnosticSourceActivities != null && this.IncludeDiagnosticSourceActivities.Count > 0)
                             {
                                 this.telemetryDiagnosticSourceListener = new TelemetryDiagnosticSourceListener(configuration, this.IncludeDiagnosticSourceActivities);
+                                this.telemetryDiagnosticSourceListener.RegisterHandler(ServiceBusDiagnosticsEventHandler.DiagnosticSourceName, new ServiceBusDiagnosticsEventHandler(configuration));
```

Comment:
registering handler for ServiceBus is required _only if_  IncludeDiagnosticSourceActivities contains ServiceBus right?

Reply:
nope, handler is registered anyway, however if there ServiceBus is disabled, handler will never ever be called - filtering of activities is done by `TelemetryDiagnosticSourceListener`

Reply:
ok